invenio_rdm_records/theme/views.py

```diff
@@ -49,7 +51,29 @@ def to_date(date_string):
         ```
     """
     assert isinstance(date_string, str)
-    return arrow.get(date_string).date()
+    date = ""
+    try:
+        date = arrow.get(date_string).date()
+    except ParserError:
+        date = date_string
+    return date
+
+
+@blueprint.app_template_filter('format_date')
+def format_date(date, format):
+    """Return a formatted Date object.
```

Comment:
```suggestion
    """Return a formatted Date string.
```

Reply:
@fenekku actually the date is of type Date/string, if it's string( thus an edtf datestring) then it doesn't format it otherwise it returns the formatted date. How should I phrase that? :)

Reply:
It returns a string in both cases, no? (but the input may be a string or a Date)

Reply:
Ah yes sorry :) You are right!